config/boards/firefly-itx-3588j

```diff
@@ -0,0 +1,21 @@
+# Rockchip RK3588j Octa core 4GB-32GB eMMC GBE HDMI HDMI-IN PCIe SATA USB3 WiFi 4G 5G
+BOARD_NAME="Firefly ITX-3588J"
+BOARDFAMILY="rockchip-rk3588"
+BOARD_MAINTAINER=""
+KERNEL_TARGET="vendor,current,edge"
```

Comment:
There is no `current` branch for `rockchip-rk3588`. Not yet at least.

My prediction is this will be with the release of Linux 6.12 at the earliest because 6.11 might become next LTS kernel which then will have enough stuff builtin to provide a decent experience for rk3588.